cmd/kubeadm/app/util/user.go

```diff
@@ -0,0 +1,15 @@
+package util
+
+import "os/user"
+
+const (
+	sudoUser = "0"
+)
+
+func IsRoot() (*user.User, bool) {
```

Comment:
I'd prefer to move this function into a separate package: `cmd/kubeadm/app/preflight`